### PR TITLE
Missing Shex files added

### PR DESCRIPTION
**Pull request by pabloalarconm**
Missinf ShEx files are added, please review

### CODE REVIEW

File: shex/sio-model/8_Consent.shex [hunk 1]
```diff
@@ -0,0 +1,26 @@
+PREFIX : <http://purl.org/ejp-rd/cde/v010/shex/>
```
Comment (rajaram5) [inline shex/sio-model/8_Consent.shex:1]: we have to use new version in the base URI. Can you please change `v010` to `v020`?

File: shex/sio-model/biobank.shex [hunk 2]
```diff
@@ -0,0 +1,25 @@
+PREFIX : <http://purl.org/ejp-rd/cde/v010/shex/>
```
Comment (rajaram5) [inline shex/sio-model/biobank.shex:1]: we have to use new version in the base URI. Can you please change `v010` to `v020`?

File: shex/sio-model/disability.shex [hunk 3]
```diff
@@ -0,0 +1,72 @@
+PREFIX : <http://purl.org/ejp-rd/cde/v010/shex/>
```
Comment (rajaram5) [inline shex/sio-model/disability.shex:1]: we have to use new version in the base URI. Can you please change `v010` to `v020`?

**Review by rajaram5 — CHANGES_REQUESTED** [review]

File: shex/sio-model/disability.shex [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  a [ snomedct:715823002 ] ;
+  sio:SIO_000008 @:percentageShape ;
+  sio:SIO_000300 xsd:decimal 
+}
+
+:percentageShape IRI {
+  a [ obo:UO_0000187 ] ;
+  sio:SIO_000300 xsd:decimal
+}
+
```
Comment (rajaram5) [inline shex/sio-model/disability.shex:63]: Can you remove the empty lines please?

File: shex/sio-model/disability.shex [hunk 5]
```diff
@@ -0,0 +1,72 @@
+PREFIX : <http://purl.org/ejp-rd/cde/v020/shex/>
+PREFIX obo: <http://purl.obolibrary.org/obo/> 
+PREFIX sio: <http://semanticscience.org/resource/>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX snomedct: <http://purl.bioontology.org/ontology/SNOMEDCT/>
```
Comment (rajaram5) [inline shex/sio-model/disability.shex:5]: PREFIX `xsd` is missing

File: shex/sio-model/disability.shex [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+:testprocessShape IRI {
+  a [ obo:OMIT_0028217 ] ;
+  sio:SIO_000230 @:questionnaire_questionShape ;
+  sio:SIO_000229 @:answerShape
+}
+
+:answerShape IRI {
+  a [ sio:SIO_000340 ] ;
+  sio:SIO_000221 obo:UO_0000033 ;
+  sio:SIO_000300 xsd:string
```
Comment (rajaram5) [inline shex/sio-model/disability.shex:43]: According to [example rdf ](https://github.com/NuriaQueralt/CDE-semantic-model/blob/develop/example-data/turtle/sio-model/disability.ttl)`xsd:string` should be `xsd:integer`

File: shex/sio-model/disability.shex [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+  sio:SIO_000642 @:answerShape ;
+  sio:SIO_000216 @:scoreShape
+}
+
+:questionnaireShape IRI {
+  a [ obo:NCIT_C130322 ] ;
+  sio:SIO_000028 @:questionnaire_questionShape
+}
+
+:questionnaire_questionShape IRI {
```
Comment (rajaram5) [inline shex/sio-model/disability.shex:29]: can you use camel case here? Replace `:questionnaire_questionShape` by `:questionnaireQuestionShape`

File: shex/sio-model/disability.shex [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+  a [ obo:NCIT_C130916 ] ;
+  sio:SIO_000300 xsd:string
+}
+
+:testprocessShape IRI {
+  a [ obo:OMIT_0028217 ] ;
+  sio:SIO_000230 @:questionnaire_questionShape ;
```
Comment (rajaram5) [inline shex/sio-model/disability.shex:36]: can you use camel case here?

File: shex/sio-model/biobank.shex [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+PREFIX obo: <http://purl.obolibrary.org/obo/> 
+PREFIX sio: <http://semanticscience.org/resource/>
+
+
+
+:specimenShape IRI {
+  a [ sio:SIO_010412 ] ;
+  sio:SIO_00022 @:roleShape
```
Comment (sxzhang1201) [inline shex/sio-model/biobank.shex:9]: I guess the property should be "sio:SIO_000228".

File: shex/sio-model/biobank.shex [hunk 10]
```diff
@@ -0,0 +1,25 @@
+PREFIX : <http://purl.org/ejp-rd/cde/v010/shex/>
+PREFIX obo: <http://purl.obolibrary.org/obo/> 
+PREFIX sio: <http://semanticscience.org/resource/>
+
```
Comment (sxzhang1201) [inline shex/sio-model/biobank.shex:4]: You lack the prefix for xsd.

File: shex/sio-model/disability.shex [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+  sio:SIO_000229 @:answerShape
+}
+
+:answerShape IRI {
+  a [ sio:SIO_000340 ] ;
+  sio:SIO_000221 obo:UO_0000033 ;
+  sio:SIO_000300 xsd:string
```
Comment (sxzhang1201) [inline shex/sio-model/disability.shex:43]: The type of literal is conflicted with the example, which uses xsd:integer.

File: shex/sio-model/disability.shex [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+
+:testprocessShape IRI {
+  a [ obo:OMIT_0028217 ] ;
+  sio:SIO_000230 @:questionnaire_questionShape ;
+  sio:SIO_000229 @:answerShape
+}
+
+:answerShape IRI {
+  a [ sio:SIO_000340 ] ;
+  sio:SIO_000221 obo:UO_0000033 ;
```
Comment (rajaram5) [inline shex/sio-model/disability.shex:42]: You missed `[]` here.  `obo:UO_0000033` should be `[obo:UO_0000033]`

**Review by rajaram5 — CHANGES_REQUESTED** [review]

File: shex/sio-model/disability.shex [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+:questionnaireShape IRI {
+  a [ obo:NCIT_C130322 ] ;
+  sio:SIO_000028 @:questionnaire_questionShape
+}
+
+:questionnaireQuestionShape IRI {
+  a [ obo:NCIT_C130916 ] ;
+  sio:SIO_000300 xsd:integer
```
Comment (rajaram5) [inline shex/sio-model/disability.shex:32]: `xsd:integer` should of type `xsd:string`

File: shex/sio-model/disability.shex [hunk 14]
```diff
@@ -0,0 +1,63 @@
...
+  a [ obo:OMIT_0005448 ] ;
+  sio:SIO_000230 @:answerShape ;
+  sio:SIO_000229 @:scoreShape ;
+  prov:wasInformedBy @:testprocessShape
+}
+
+:scoreShape IRI {
+  a [ snomedct:715823002 ] ;
+  sio:SIO_000008 @:percentageShape ;
+  sio:SIO_000300 xsd:integer
```
Comment (rajaram5) [inline shex/sio-model/disability.shex:57]: `xsd:integer` should be of type `xsd:decimal`

**Review by rajaram5 — CHANGES_REQUESTED** [review]
Minor issues need to be addressed

**Comment by rajaram5** [comment]
Looks good @pabloalarconm . I will merge it. Thanks

**Pull request merged**